robottelo/config/__init__.py

```diff
@@ -11,6 +12,16 @@
 
 logger = logging.getLogger('robottelo.config')
 
+
+def get_project_root():
+    """Return the path to the Robottelo project root directory.
+
+    :return: A directory path.
+    :rtype: str
+    """
+    return os.path.realpath(os.path.join(os.path.dirname(__file__), os.pardir, os.pardir))
```

Comment:
if you used pathlib's Path class, you can do the same with the below. that is unless I'm confused about exactly what's going on here.
```suggestion
    return str(Path().absolute())
```
if you need the absolute path of this file `str(Path(__file__).absolute())`

Reply:
I'm down to use pathlib throughout this.

Reply:
Here it is resolving a grand-parent directory, which should be the root robottelo directory relative to the file.